yb-voyager/src/srcdb/pg_dump_export_data.go

```diff
@@ -33,22 +36,25 @@ import (
 func pgdumpExportDataOffline(ctx context.Context, source *Source, connectionUri string, exportDir string, tableList []*sqlname.SourceName, quitChan chan bool, exportDataStart chan bool, exportSuccessChan chan bool) {
 	defer utils.WaitGroup.Done()
 
-	dataDirPath := exportDir + "/data"
-
-	tableListPatterns := createTableListPatterns(tableList)
-
-	// Using pgdump for exporting data in directory format.
-
 	pgDumpPath, err := GetAbsPathOfPGCommand("pg_dump")
 	if err != nil {
 		utils.ErrExit("could not get absolute path of pg_dump command: %v", err)
 	}
 
-	pgDumpArgs := fmt.Sprintf(`--no-blobs --data-only --no-owner --compress=0 %s -Fd --file %s --jobs %d --no-privileges --no-tablespaces --load-via-partition-root`,
-		tableListPatterns, dataDirPath, source.NumConnections)
+	pgDumpArgs.DataDirPath = filepath.Join(exportDir, "data")
+	pgDumpArgs.TablesListPattern = createTableListPatterns(tableList)
+	pgDumpArgs.ParallelJobs = strconv.Itoa(source.NumConnections)
+	pgDumpArgs.DataFormat = "directory"
+
+	args := source.getPgDumpDataArgsFromFile()
+	if args == "" {
```

Comment:
Under what circumstances would be get empty args? 
Even if the file is not present, we can have a default conf file embedded into go, and parse that, no?

That would be more consistent than having a different set of default args in code. Basically, one single source of args is what I'm getting at.

Reply:
yeah agreed, will make the change